src/components/Banner/BaseAnnouncementBanner/index.tsx

```diff
@@ -1,105 +0,0 @@
-import { Trans } from '@lingui/macro'
```

Comment:
nit: I think removal of the Base banner would ideally happen in a separate PR with its migration